src/Symfony/Bridge/Doctrine/Tests/Validator/DoctrineLoaderTest.php

```diff
@@ -175,9 +187,33 @@ public function testClassValidator(bool $expected, string $classValidatorRegexp
     public function regexpProvider()
     {
         return [
-            [false, null],
```

Comment:
I don't agree with this change. It causes a regression (see my comment at https://github.com/symfony/symfony/pull/33828#pullrequestreview-297993708).

Reply:
And anyway it must not be in this PR. Reverting here, let's discuss on #33828

Reply:
Just reverting it isn't possible actually. We need to decide what behavior we want.

Reply:
Should be resolved in #33828 now.